Storage/Storage/Model/TaxClass+CoreDataProperties.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+import CoreData
+
+
+extension TaxClass {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<TaxClass> {
+        return NSFetchRequest<TaxClass>(entityName: "TaxClass")
+    }
+
+    @NSManaged public var name: String?
+    @NSManaged public var slug: String?
```

Comment:
same as in the Networking model, might make sense to make this or both properties non-optional?